### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by raynol-dsouza**
Comments are in the commits.

### CODE REVIEW

File: pyiron_contrib/__init__.py [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
-from pyiron_contrib.protocol.compound.qmmm import ProtocolQMMM
-from pyiron_contrib.protocol.compound.tild import ProtocolHarmonicTILD, ProtocolHarmonicTILDParallel
+from pyiron import Project
+from pyiron.base.job.jobtype import JOB_CLASS_DICT
 
-# protocol is a magic class after this one we have to Register
-# from pyiron_contrib.protocol.utils.types import PyironJobTypeRegistryMetaType
-# PyironJobTypeRegistryMetaType.inject_dynamic_types()
-__all__ = [
-    'ImageJob',
```
Comment (liamhuber) [inline pyiron_contrib/__init__.py:16]: Please keep ImageJob in the new indexing

File: pyiron_contrib/__init__.py [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
-]
+# Make classes available for new pyiron version
+JOB_CLASS_DICT['ProtocolMinimize'] = 'pyiron_contrib.protocol.compound.minimize'
+JOB_CLASS_DICT['ProtocolMD'] = 'pyiron_contrib.protocol.compound.molecular_dynamics'
+JOB_CLASS_DICT['ProtocolNEB'] = 'pyiron_contrib.protocol.compound.nudged_elastic_band'
+JOB_CLASS_DICT['ProtocolQMMM'] = 'pyiron_contrib.protocol.compound.qmmm'
+JOB_CLASS_DICT['ProtocolHarmonicTILD'] = 'pyiron_contrib.protocol.compound.tild'
+JOB_CLASS_DICT['ProtocolHarmonicTILDParallel'] = 'pyiron_contrib.protocol.compound.tild'
+
+# Backwards compatibility
```
Comment (jan-janssen) [inline pyiron_contrib/__init__.py:12]: Should not this part be in `pyiron_mpie` ?

**Review by liamhuber — CHANGES_REQUESTED** [review]
Needs ImageJob included, otherwise looks good!

File: pyiron_contrib/__init__.py [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
-from pyiron_contrib.protocol.compound.minimize import ProtocolMinimize
-from pyiron_contrib.protocol.compound.molecular_dynamics import ProtocolMD
-from pyiron_contrib.protocol.compound.nudged_elastic_band import ProtocolNEB
-# from pyiron_contrib.protocol.compound.tild import HarmonicTILD, VacancyTILD
-# from pyiron_contrib.protocol.compound.finite_temperature_string import StringRelaxation, VirtualWork, \
-#     Milestoning, VirtualWorkParallel, VirtualWorkSerial, VirtualWorkFullStep
-from pyiron_contrib.protocol.compound.qmmm import ProtocolQMMM
-from pyiron_contrib.protocol.compound.tild import ProtocolHarmonicTILD, ProtocolHarmonicTILDParallel
+from pyiron import Project
+from pyiron.base.job.jobtype import JOB_CLASS_DICT
```
Comment (jan-janssen) [inline pyiron_contrib/__init__.py:2]: Please be aware that `pyiron.base` changes to `pyiron_base` - most likely on Friday https://github.com/pyiron/pyiron/pull/952

File: pyiron_contrib/__init__.py (hunk 2 above)
Comment (raynol-dsouza) [reply pyiron_contrib/__init__.py:12]: I agree. At times I only import `pyiron_contrib` and not `pyiron_mpie`. This kind-of helps in those circumstances. However, I can remove it. Shall I?

File: pyiron_contrib/__init__.py (hunk 3 above)
Comment (raynol-dsouza) [reply pyiron_contrib/__init__.py:2]: We can leave this pull request open till then, I reckon? Or I can open an issue on `pyiron_contrib` as a reminder?

File: pyiron_contrib/__init__.py (hunk 2 above)
Comment (jan-janssen) [reply pyiron_contrib/__init__.py:12]: I would prefer that, because people who do not have access to `pyiron_mpie` might be confused.

Comment (raynol-dsouza) [reply pyiron_contrib/__init__.py:13]: Sure. I shall remove it.

File: pyiron_contrib/__init__.py (hunk 3 above)
Comment (jan-janssen) [reply pyiron_contrib/__init__.py:2]: Either way is fine for me. As it is a major change I wanted to announce it in this weeks pyiron meeting before merging it.

Comment (liamhuber) [reply pyiron_contrib/__init__.py:2]: I will merge and close now as this is relevant for UBC. We'll make a fresh pull once the change is in place

**Review by liamhuber — APPROVED** [review]

**Pull request merged**